test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1939,6 +1939,16 @@ public void scans() throws Exception {
     ts.exec("deletetable -f t");
   }
 
+  @Test
+  public void scansWithColon() throws Exception {
+    ts.exec("createtable twithcolontest");
+    make10WithColon();
+    ts.exec("scan -r row0 -cf c:f", true, "value");
+    String result = ts.exec("scan -b row1 -cf c:f  -cq col1 -e row1");
+    assertEquals(2, result.split("\n").length);
```

Comment:
Could add a test case for if a user adds the `-cq` option without adding a `-cf` and one where they also add the `-c` option.

Reply:
sure